x-pack/libbeat/management/api/client.go

```diff
@@ -60,18 +59,23 @@ func NewClient(cfg *kibana.ClientConfig) (*Client, error) {
 
 // do a request to the API and unmarshall the message, error if anything fails
 func (c *Client) request(method, extraPath string,
-	params common.MapStr, headers http.Header, message interface{}) (int, error) {
+	body interface{}, headers http.Header, message interface{}) (int, error) {
```

Comment:
nit: i'd prefer 'headers' before 'body' :)

What is the difference between 'body' and 'message' ?

Reply:
Oh, I see. `message` is the response structure. Maybe we should call it resp then? It should also be clear the it must be a pointer.

Reply:
I can change the order and naming, +1 :)

Reply:
I will do the ordering in another PR, that will break my other in flight PR :(